File: 프로그래머스/1/176963. 추억 점수/추억 점수.py
def solution(name, yearning, photo):
    answer = []
    for i in range(len(photo)):
        temp = 0
        for j in range(len(photo[i])):
            if photo[i][j] in name:
            	temp += yearning[name.index(photo[i][j])]
            else:
                continue
        answer.append(temp)
            
    return answer

File: 프로그래머스/1/176963. 추억 점수/추억 점수.js
function solution(name, yearning, photo) {
    var answer = [];
    for(let i=0; i<photo.length; i++){
        let counter = 0;
        for(let j=0; j<photo[i].length;j++){
            if(name.includes(photo[i][j])){
                counter += yearning[name.indexOf(photo[i][j])];
            }
        }
        answer.push(counter);
    }
    return answer;
}